### PR TITLE
Adding changes for PyTorch 1.4.0 DLC

### PR DESCRIPTION
**Pull request by abhinavs95**
*Description of changes:*

1. Adds PyTorch 1.4.0
2. Adds Torchvision 0.5.0
3. Upgrades to Cuda 10.1 for PyTorch 1.4.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docker/1.4.0/py2/Dockerfile.cpu [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ && pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
+ && pip install mxnet-model-server==$MMS_VERSION
+
+RUN useradd -m model-server \
+ && mkdir -p /home/model-server/tmp \
+ && chown -R model-server /home/model-server
+
+COPY mms-entrypoint.py /usr/local/bin/dockerd-entrypoint.py
+COPY config.properties /home/model-server
+COPY deep_learning_container.py /usr/local/bin/deep_learning_container.py
```
Comment (arjkesh) [inline docker/1.4.0/py2/Dockerfile.cpu]: Similarly to training, this needs to be removed in all places

**Comment by codewithsk** [comment]
@abhinavs95 Looks like github is showing this branch is out of date with master. Would you be able to rebase? Thanks.

File: docker/1.4.0/py2/Dockerfile.cpu [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    conda-forge \
+    awscli \
+    opencv==4.0.1 \
+ && conda install -y \
+    scikit-learn==0.20.3 \
+    pandas==0.24.2 \
+    pillow==6.2.1 \
+    h5py==2.9.0 \
+    requests==2.22.0 \
+ && conda install \
```
Comment (codewithsk) [inline docker/1.4.0/py2/Dockerfile.cpu]: Packages from line 38~63 seem to be out of date. Is updating any of these packages causing broken requirements? 

I took a cursory look and found these packages to be outdated. 
cython - 0.29.12->0.29.14
ipython - 5.8.0 -> 7.12.0
mkl-include - 2019.4 -> 2020
mkl - 2019.4 -> 2020
numpy - 1.16.4 -> 1.18.1

Comment (codewithsk) [reply docker/1.4.0/py2/Dockerfile.cpu]: This applies to all four dockerfiles.

**Comment by codewithsk** [comment]
@abhinavs95 Looks like there's a merge conflict. Can you get that resolved please?

**Pull request merged**